src/AutosuggestContainer.js

```diff
@@ -71,6 +71,7 @@ export default class AutosuggestContainer extends Component {
     getSectionSuggestions: PropTypes.func,
     focusInputOnSuggestionClick: PropTypes.bool,
     inputRef: PropTypes.func,
+    componentRef: PropTypes.func,
```

Comment:
Please set a default value for this prop

Reply:
Done, thanks.